test/bigtest/tests/checkIn-details-modal-test.js

```diff
@@ -0,0 +1,86 @@
+import { describe, beforeEach, it } from '@bigtest/mocha';
+import { expect } from 'chai';
+
+import setupApplication from '../helpers/setup-application';
+import CheckInDetailsModal from '../interactors/checkIn-details-modal';
+import CheckInItemsPage from '../interactors/check-in-items-page';
+import { WORKFLOW_STATUS } from '../../../src/components/PurchaseOrder/Summary/FieldWorkflowStatus';
+import { PHYSICAL } from '../../../src/components/POLine/const';
+import { PIECE_STATUS_EXPECTED } from '../../../src/components/Receiving/const';
+import { ORDERS_API } from '../../../src/components/Utils/api';
+
+const RECEIVING_LIST_COUNT = 10;
+const TEST_BARCODE = 'test_barcode';
+
+describe('CheckIn Details Modal', () => {
+  setupApplication();
+
+  let order = null;
+  let line = null;
+  const modal = new CheckInDetailsModal();
+  const page = new CheckInItemsPage();
+
+  beforeEach(async function () {
+    order = await this.server.create('order', {
```

Comment:
it looks like this a sync operation, no need an await

Reply:
Fixed